app/src/main/java/co/touchlab/kampstarter/android/BreedViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package co.touchlab.kampstarter.android
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import co.touchlab.kampstarter.db.Breed
+import co.touchlab.kampstarter.models.BreedModel
+import kotlinx.coroutines.InternalCoroutinesApi
+import androidx.lifecycle.viewModelScope
+import co.touchlab.kampstarter.models.ItemDataSummary
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+@InternalCoroutinesApi
+class BreedViewModel(app: Application) : AndroidViewModel(app) {
```

Comment:
Can use `ViewModel` instead of `AndroidViewModel` and then there's no need to pass the application through

Reply:
Done